### PR TITLE
chore(Integrate CircleCi and Coveralls badges ):testing environment 184637125

### PR DESCRIPTION
**Pull request by divinecharlotte**
### What does this PR do?

- Integrate CircleCI with the readme badge
- Integrate Coeralls with readme badge

### Description of Task to be completed?

1. set up local tests
2. integrate circle-Ci: This is required to automatically run tests and other preliminary build steps required to ship the next iteration of the product
3. Integrate Coeralls: This is to ensure we are properly informed just how well our code base is tested which in turn provides quality assurance for the product

### How should this be manually tested?

**Installation**
**1. Clone the repository by running** 
```
git clone https://github.com/atlp-rwanda/destructors-ec-bn.git
```
**2. Navigate to the project directory by running cd destructors-ec-bn.**
```
cd destructors-ec-bn
```
**3. Install the necessary dependencies by running npm install.**
```
npm install
```
**4. Run the development server by running npm run dev. This will start the server using Nodemon, which will automatically reload the server when changes are made to the code.**
```
npm run dev
```
**5. Run the tests by running npm run test** 
```
npm run test
```
### Any background context you want to provide?

- none

### What are the relevant pivotal tracker/Trello stories?

- Tracker: 184637125
- Tracker: 184637124
### Screenshots (if appropriate)

- none

### Questions:

- none

### CODE REVIEW

**Review by mjeanbosco19 — COMMENTED** [review]
You did a great job @divinecharlotte keep it up!!!  
I noticed that your Pull Request is missing the task ID at the beginning of the name. As a reminder, we require that all our Pull Requests follow the naming convention of starting with the task ID and then a brief description name.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+jobs:
+  build:
+    docker:
+      - image: node:16.17.0
+    environment: 
+      repo_token: REPO_TOKEN
```
Comment (Abdulkeza) [inline .circleci/config.yml]: Have you added this Environment variable in github secretes and it's value?

**Review by Abdulkeza — DISMISSED** [review]

**Review by Abdulkeza — CHANGES_REQUESTED** [review]
add one test for testing

**Review by mjeanbosco19 — CHANGES_REQUESTED** [review]
You should add coverage directory to .gitignore.  As reminder, you should add Label on your PR like WIP or Ready for Review.

**Review by mjeanbosco19 — APPROVED** [review]
Amazing!!! Keep it up @divinecharlotte

**Review by leandreAlly — APPROVED** [review]

**Review by istafrica-dennw — CHANGES_REQUESTED** [review]
1. There is no build
2. there is coverage 

on the budges

File: coverage/clover.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<coverage generated="1678368858788" clover="3.2.0">
+  <project timestamp="1678368858788" name="All files">
+    <metrics statements="1" coveredstatements="1" conditionals="0" coveredconditionals="0" methods="1" coveredmethods="1" elements="2" coveredelements="2" complexity="0" loc="1" ncloc="1" packages="1" files="1" classes="1"/>
+    <file name="sum.js" path="C:\Users\Administrator\Desktop\atlp projects\destructors-ec-bn\src\sum.js">
+      <metrics statements="1" coveredstatements="1" conditionals="0" coveredconditionals="0" methods="1" coveredmethods="1"/>
+      <line num="2" count="1" type="stmt"/>
+    </file>
+  </project>
+</coverage>
```
Comment (Abdulkeza) [inline coverage/clover.xml]: @divinecharlotte , remember to put coverage folder in **gitignore**

ex: coverage

**Review by Abdulkeza — CHANGES_REQUESTED** [review]
remember to add coverage in **gitignore**

File: .babelrc [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
+<<<<<<< HEAD
   "presets": [
       "@babel/preset-env"
   ]
```
Comment (Abdulkeza) [inline .babelrc]: What it this @divinecharlotte ?

File: .babelrc [hunk 4]
```diff
@@ -7,4 +8,12 @@
...
\ No newline at end of file
+  
+=======
+    "env": {
+      "test": {
+        "plugins": ["@babel/plugin-transform-modules-commonjs"]
+      }
+    }
+  }
+>>>>>>> 3c62b51 (add a single test to check the environment)
```
Comment (Abdulkeza) [inline .babelrc]: please have you resolved conflicts in the perfect way?

**Review by Abdulkeza — CHANGES_REQUESTED** [review]
Please make sure the conflicts are resolved correctly

**Review by Abdulkeza — APPROVED** [review]

**Review by Abdulkeza — APPROVED** [review]

**Pull request merged**